subsys/net/lib/http/http_client.c

```diff
@@ -0,0 +1,484 @@
+/*
+ * Copyright (c) 2019 Unisoc Technologies INC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_http_client, CONFIG_HTTP_CLIENT_LOG_LEVEL);
+
+#include <zephyr.h>
+#include <string.h>
+#include <strings.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <version.h>
+
+#include <sys/time.h>
+#include <net/net_core.h>
+#include <net/net_ip.h>
+#include <net/socket.h>
+#include <net/http_parser.h>
+#include <kernel.h>
+
+#if defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS)
+#include <net/tls_credentials.h>
+#include "ca_certificate.h"
+#endif
+
+#include "../../ip/net_private.h"
+#include <net/http_client.h>
+
+#define HTTP_HOST          "Host"
+#define HTTP_CONTENT_TYPE  "Content-Type"
+#define HTTP_CONTENT_LEN   "Content-Length"
+
+#define MAX_NUM_DIGITS	16
+
+
+static int on_status(struct http_parser *parser,
+					const char *at, size_t length)
+{
+	struct http_ctx *ctx = CONTAINER_OF(parser,
+					    struct http_ctx,
+					    parser);
+
+	if (length > (HTTP_STATUS_STR_SIZE - 1)) {
+		NET_ERR("HTTP response status error, status=%s", at);
+		return -1;
+	}
+	memcpy(ctx->rsp.http_status, at, length);
```

Comment:
always have an empty line after a '}' (unless  it's another '}' etc..)

Apply that everywhere